<launch>
  <arg name="input_point_topic" default="/points_raw" />
  <arg name="input_map_topic" default="/points_map" />
  <arg name="output_match_topic" default="/points_ground" />
  <arg name="output_unmatch_topic" default="/points_no_ground" />

  <node pkg="points_preprocessor" type="compare_map_filter" name="compare_map_filter">
    <remap from="/points_raw" to="$(arg input_point_topic)"/>
    <remap from="/points_map" to="$(arg input_map_topic)"/>
    <remap from="/points_ground" to="$(arg output_match_topic)"/>
    <remap from="/points_no_ground" to="$(arg output_unmatch_topic)"/>
  </node>

</launch>
